screenshot-app/pages/api/screenshot.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  const options = {
+    method: "GET",
+    url: "https://website-screenshot6.p.rapidapi.com/screenshot",
+    params: {
+      url: req.query.url,
+      width: "1920",
+      height: "1080",
+      fullscreen: "false",
+    },
+    headers: {
+      "x-rapidapi-host": "website-screenshot6.p.rapidapi.com",
```

Comment:
I tried to use the app multiple times. But, the API is returning the 504 status code. Please check.

Reply:
It is working on my end, although some websites which probably take longer to load are causing this error. I will display the error message.